### PR TITLE
Minor link fixes

### PR DESCRIPTION
**Pull request by ivernus**
Fixed several links to move to new URL (instead of redirecting via another page), modified link for help wanted issues.

Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).

When filling in a section please remove the help text and the above text.

#### Summary
Minor changes to readme

#### Ticket Link
n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,15 +28,15 @@ Install on your own machine:
 Get Involved:
 
 - [Contribute Code](http://docs.mattermost.com/developer/contribution-guide.html)
-- [Find "Help Wanted" projects](https://mattermost.atlassian.net/issues/?filter=10101)
+- [Find "Help Wanted" projects](https://mattermost.atlassian.net/browse/PLT-7805?filter=10101&jql=status%20in%20(Open%2C%20Reopened)%20AND%20labels%20in%20(accepting-pull-requests%2C%20help-wanted))
```
Comment (grundleborg) [inline README.md]: This doesn't look right.

Comment (ivernus) [reply README.md]: ya it was quite difficult to make the tags work properly, is there a "short url" format that can be exported from jira?

**Comment by ivernus** [comment]
Thanks @jasonblais ill make the update pointing to that instead

**Comment by jasonblais** [comment]
Thanks! The URL will also be simpler than Jira's which can sometimes lead to issues (as you pointed out above)

**Comment by jasonblais** [comment]
Thanks @ivernus! :)

**Pull request merged**